### ISSUE TITLE
App Idea: Graphene Django

### ISSUE THREAD

**Issue by zbyte64** [metakermit/generator-django-rest#6]
Would like to add an option during generation to add Graphene support. Doing so would add a schema.py with a query and mutation imported from django-graph-auth, and mount the schema in the urls. Cool with this and any advice?

**Comment by zbyte64** [comment]
Was actually going to open a Django 2.0 issue; python 2 won't be supported after 2020.

**Comment by metakermit** [comment]
OK, I've just released version 0.7.1 which includes both Pipenv & Django 2.0.4. I've tested the basic workflow and `docker-compose up` which both seems to work. There will probably still be some rough parts that will need fixing, though.

I've also started keeping a [changelog](https://github.com/metakermit/generator-django-rest/blob/master/CHANGELOG.md) to make it easier to keep track of progress.
